scripts/fix-markdown-strings.py

```diff
@@ -22,6 +22,14 @@
     "## If you want to know more": "## ℹ️ If you want to know more",
 }
 
+ADMONITION_TITLE = "These are the key outcomes of this assessment"
+
+QA_STATEMENT_FORMAT = {
```

Comment:
"l0", "l1", makes me think we could use just a list/tuple rather than a dict? `LINES = ["...", "...", "..."]`

l0 = LINES[0]